module6/intermission/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: Mod 6 Intermission Work
+---
+
+Mod 6 is all about preparing you for life after Turing.  During the module, we will set up a schedule that will help give structure to our Job Hunt.  This schedule will include job hunt activities and continued technical growth.  We will guide those efforts during the module, but after graduation, it will be up to you to find ways to continue your technical growth.
+
+One way that you can continue your technical growth is by completing [certifications](https://www.google.com/search?q=list+of+certifications+for+developers).  These certifications are a way to document your learning, and continue to improve.
+
+### A Good First Cert
+
+During intermission, we would like you to earn the [Foundational C# with Microsoft](https://www.freecodecamp.org/learn/foundational-c-sharp-with-microsoft/) through FreeCodeCamp.  This cert was put together in coordination with the .NET team!
+
+In order to complete the certification, you will need accounts with:
+* [FreeCodeCamp](https://www.freecodecamp.org/)
+* [Microsoft](https://account.microsoft.com/account/manage-my-account)
+
+As you work through this certification, much of the content will feel like review.  We encourage you to dive in and read through all of the documentation - you never know where you might find new information, or a different way to think about a problem!
+
+✅When finished, send your instructors a link to your certification!
```

Comment:
This seems like a great plan! My only hesitation is about the amount of time it would take to get that certification. I think maybe we time cap it at 8 hours. And this is something they could finish over the winter break too!!

Reply:
Ooh - good call on continuing over winter break!!